docs/getting-started.md

```diff
@@ -44,74 +43,138 @@ Verify that the image is available in your account and remember the correspondin
     $ doctl compute image list-user
 
 
-## Cluster Creation
-
-> We assume you already have a running a management cluster
+## Initialize the management cluster
 
 ```bash
-export CLUSTER_NAME=capdo-quickstart # change this name as you prefer.
-export MACHINE_IMAGE=<image-id> # created in the step above.
+$ export DO_B64ENCODED_CREDENTIALS="$(echo -n "${DIGITALOCEAN_ACCESS_TOKEN}" | base64 | tr -d '\n')"
+
+# Initialize a management cluster with digitalocean infrastructure provider.
+$ clusterctl init --infrastructure digitalocean
 ```
 
-For the purpose of this tutorial, we’ll name our cluster `capdo-quickstart`.
+The output will be similar to this:
 
-Generate examples files. (Several parameters such as the target Kubernetes version or the machine types can be customized by setting specific environment variables; see the variable definitions in the `examples/generate.sh` file for the list of available options.)
+```bash
+Fetching providers
+Installing cert-manager Version="v0.16.1"
+Waiting for cert-manager to be available...
+Installing Provider="cluster-api" Version="v0.3.11" TargetNamespace="capi-system"
+Installing Provider="bootstrap-kubeadm" Version="v0.3.11" TargetNamespace="capi-kubeadm-bootstrap-system"
+Installing Provider="control-plane-kubeadm" Version="v0.3.11" TargetNamespace="capi-kubeadm-control-plane-system"
+Installing Provider="infrastructure-digitalocean" Version="v0.4.0" TargetNamespace="capdo-system"
+
+Your management cluster has been initialized successfully!
+
+You can now create your first workload cluster by running the following:
+
+  clusterctl config cluster [name] --kubernetes-version [version] | kubectl apply -f -
 
-```
-make generate-examples
 ```
 
-Install core-component(CAPI & CABPK) and provider-component (CAPDO)
+## Creating a workload cluster
 
+Setting up environment variable
+
+```bash
+$ export DO_REGION=<region>
+$ export DO_SSH_KEY_FINGERPRINT=<your-ssh-key-fingerprint>
+$ export DO_CONTROL_PLANE_MACHINE_TYPE=<droplet-size>
+$ export DO_CONTROL_PLANE_MACHINE_IMAGE=<image-id> # created in the step above.
+$ export DO_NODE_MACHINE_TYPE=<droplet-size>
+$ export DO_NODE_MACHINE_IMAGE=<image-id> # created in the step above.
 ```
-kubectl apply -f examples/_out/core-components.yaml
-kubectl apply -f examples/_out/provider-components.yaml
+Generate templates for creating workload clusters.
+
+```bash
+$ clusterctl config cluster capdo-quickstart \
+    --infrastructure digitalocean \
+    --kubernetes-version v1.17.11 \
+    --control-plane-machine-count 1 \
+    --worker-machine-count 3 > capdo-quickstart-cluster.yaml
 ```
 
-Create cluster and control plane machine
+*You may need to inspect and make some changes of generated template.*
```

Comment:
grammar nits: 
* `s/of/to`
* `generated template` -> `the generated template`

specifically, we want to make changes `to` the specific object and `the` is needed before the noun, `generated template`. 

The change should look like this:
`make some changes of generated template.` should be `make some changes to the generated template.`